.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
I suggest you add a `node_modules/ ` folder on the .gitignore so that node files that maybe added to the project  (can be added by accident while setting up Linters). are not pushed to github.